.stylelintrc.js

```diff
@@ -6,8 +6,17 @@
  */
 
 module.exports = {
+  extends: ['stylelint-config-recommended', 'stylelint-config-prettier'],
   plugins: ['stylelint-copyright'],
   rules: {
     'docusaurus/copyright-header': true,
+    'selector-pseudo-class-no-unknown': [
+      true,
+      {
+        ignorePseudoClasses: ['global'],
```

Comment:
Not sure if this pseudo is still supported
```
packages/docusaurus/src/client/baseUrlIssueBanner/styles.module.css
 8:1  ✖  Unexpected unknown pseudo-class selector ":global"  selector-pseudo-class-no-unknown

packages/docusaurus/lib/client/baseUrlIssueBanner/styles.module.css
 8:1  ✖  Unexpected unknown pseudo-class selector ":global"  selector-pseudo-class-no-unknown

packages/docusaurus-plugin-pwa/lib/theme/PwaReloadPopup/styles.module.css
 36:18  ✖  Unexpected unknown pseudo-class selector ":global"  selector-pseudo-class-no-unknown

packages/docusaurus-plugin-pwa/src/theme/PwaReloadPopup/styles.module.css
 36:18  ✖  Unexpected unknown pseudo-class selector ":global"  selector-pseudo-class-no-unknown

packages/docusaurus-theme-classic/src/theme/DocPage/styles.module.css
 13:1  ✖  Unexpected unknown pseudo-class selector ":global"  selector-pseudo-class-no-unknown

packages/docusaurus-theme-classic/lib/theme/DocPage/styles.module.css
 13:1  ✖  Unexpected unknown pseudo-class selector ":global"  selector-pseudo-class-no-unknown

packages/docusaurus-theme-classic/lib-next/theme/DocPage/styles.module.css
 13:1  ✖  Unexpected unknown pseudo-class selector ":global"  selector-pseudo-class-no-unknown

packages/docusaurus-theme-search-algolia/lib/theme/SearchPage/styles.module.css
 115:1  ✖  Unexpected unknown pseudo-class selector ":global"  selector-pseudo-class-no-unknown

packages/docusaurus-theme-search-algolia/src/theme/SearchPage/styles.module.css
 115:1  ✖  Unexpected unknown pseudo-class selector ":global"  selector-pseudo-class-no-unknown
```

Reply:
The `:global` is a CSS modules feature. It prevents a class name from being hashed/namespaced/